_pages/en_US/installing-boot9strap-(kartdlphax).txt

```diff
@@ -65,23 +59,24 @@ In this section, you will set up your source 3DS (the 3DS with custom firmware)
   - Create the `plugins` and `00040000...` folders if they do not already exist
 1. Eject the SD card and put it in the **source 3DS**
 
-#### Section III - Prep Work (target 3DS)
+#### Section II - Prep Work (target 3DS)
 
-In this section, you will copy the files needed to trigger the unSAFE_MODE exploit onto your target 3DS (the 3DS that you are trying to modify)'s SD card.
+In this section, you will set up the files that the target 3DS will need to install custom firmware.
 
 1. Insert the SD card of your **target 3DS** in your computer
 1. Copy `boot.firm` and `boot.3dsx` from the standard Luma3DS `.zip` to the root of your SD card
 1. Create a folder named `boot9strap` on the root of your SD card
 1. Copy `boot9strap.firm` and `boot9strap.firm.sha` from the RELEASE `.zip` to the `/boot9strap/` folder on your SD card
 1. Copy `SafeB9SInstaller.bin` from the SafeB9SInstaller `.zip` to the root of your SD card
-1. Copy `usm.bin` from the RELEASE `.zip` to the root of your SD card
-1. Create a folder called `3ds` on the root of your SD card
-1. Copy `slotTool.3dsx` from the `slotTool` folder inside the RELEASE `.zip` to the `/3ds/` folder on your SD card
+1. Create a folder named `3ds` on the root of your SD card if it does not already exist
+  * This folder stores homebrew applications and data; it is different from the `Nintendo 3DS` folder that the console automatically generates
+1. Copy the `nimhaxdsp` folder from the nimdsphax`.zip` to the `/3ds/` folder on your SD card
+1. Copy `menuhax_manager.3dsx` from the menuhax `.zip` to the `/3ds/` folder on your SD card
```

Comment:
I believe we also need to copy appropriate Launcher.dat from menuhax .zip

Reply:
This probably is the case, though it should be noted that https://3ds.hacks.guide/installing-boot9strap-(menuhax) and https://3ds.hacks.guide/installing-boot9strap-(menuhax-legacy) seem to miss this step as well.

Reply:
Because the copy step is done in Bannerbomb3.